api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupActiveOnCareList.java

```diff
@@ -165,6 +166,14 @@ public ReportDefinition constructReportDefinition() {
         addColumn(dsd,"latestViralLoad",hivPatientData.getViralLoadByEndDate());
         addColumn(dsd,"latestVLQualitative",hivPatientData.getVLQualitativeByEndDate());
         addColumn(dsd,"returnVisitDate",hivPatientData.getLastReturnDateByEndDate());
+        addColumn(dsd,"model",hivPatientData.getDSDMModel());
+        dsd.addColumn("DSDM Date",new DSDMModelDataDefinition(), "", df.getDateEnrolledConverter());
+        addColumn(dsd,"Hep-B Date",hivPatientData.getLastHepBScreeningDate());
+        addColumn(dsd,"Hep-B Result",hivPatientData.getLastHepBScreeningResult());
+        addColumn(dsd,"Hep-C Date",hivPatientData.getLastHepCScreeningDate());
+        addColumn(dsd,"Hep-C Result",hivPatientData.getLastHepCScreeningResult());
+        addColumn(dsd,"TPT Start Date",hivPatientData.getTPTInitiationDate());
+        addColumn(dsd,"TPT End Date",hivPatientData.getTPTCompletionDate());
 
 
 
```

Comment:
@dbaluku Increase the version of this report so that it can be deployed

Reply:
Done